src/main/java/de/tum/in/www1/artemis/web/rest/dto/SuspiciousExamSessionsDTO.java

```diff
@@ -0,0 +1,6 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import java.util.Set;
+
+public record SuspiciousExamSessionsDTO(Set<ExamSessionDTO> examSessions) {
```

Comment:
you could add `@JsonInclude(JsonInclude.Include.NON_EMPTY)` to avoid sending an empty set.